docs/getting_started/azure.md

```diff
@@ -0,0 +1,214 @@
+the remaining items are tracked in
+[#10412](https://github.com/kubernetes/kops/issues/10412).
+
+# Create Creation Steps
+
+## Step 1. Install Azure CLI
+
+First, install Azure CLI.
+
+```bash
+$ curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+```
+
+Then type the following command to login to Azure. This will redirect
+you to the browser login.
+
+```bash
+$ az login
+
+...
+
+You have logged in. Now let us find all the subscriptions to which you have access...
+[
+  {
+    "cloudName": "AzureCloud",
+    "homeTenantId": "76253...",
+    "id": "7e232...",
+    "isDefault": true,
+    "managedByTenants": [],
+    "name": "Your name...",
+    "state": "Enabled",
+    "tenantId": "76253...",
+    "user": {
+      "name": "...",
+      "type": "user"
+    }
+  },
+ ...
+]
+```
+
+One Azure account has one or more than one “subscription”, which
+serves as a single billing unit for Azure resources. Set the env var
+`AZURE_SUBSCRIPTION_ID` to the ID of the subscription you want to
+use.
+
+```bash
+$ export AZURE_SUBSCRIPTION_ID=7e232...
+```
+
+## Step 2. Create a Container in Azure Blob
+
+Next, create a container in Azure Blob storage, which is S3 equivalent service in Azure, to store kOps cluster configs.
+
+First, you need to create a resource group, which provides an isolated
+namespace for resources.
+
+```bash
+$ az group create --name kops-test --location eastus
+{
+  "id": "/subscriptions/7e232.../resourceGroups/kops-test",
+  "location": "eastus",
+  "managedBy": null,
+  "name": "kops-test",
+  "properties": {
+    "provisioningState": "Succeeded"
+  },
+  "tags": null,
+  "type": "Microsoft.Resources/resourceGroups"
+}
+```
+
+Then create a storage account for the resource group. The storage
+account provides an isolated namespace for all storage resources. The
+name must be unique among across multiple different Azure accounts.
+
+```bash
+$ az storage account create --name kopstest --resource-group kops-test
+```
+
+Set the env var `AZURE_STORAGE_ACCOUNT` to the storage account name for later use.
+
+```bash
+$ export AZURE_STORAGE_ACCOUNT=kopstest
+```
+
+Get an access key of the account and set it in env var `AZURE_STORAGE_KEY` for later use.
+
+```bash
+$ az storage account keys list --account-name kopstest
+[
+  {
+    "keyName": "key1",
+    "permissions": "Full",
+    "value": "RHWWn..."
+  },
+  {
+    "keyName": "key2",
+    "permissions": "Full",
+    "value": "..."
+  }
+
+]
+
+$ export AZURE_STORAGE_KEY="RHWWn...“
+```
+
+
+Then create a container.
+
+```bash
+$ az storage container create --name cluster-configs
+{
+  "created": true
+}
+```
+
+You can confirm that the container has been successfully created from
+Storage Exporter or by typing “az storage container list”.
+
+```bash
+$ az storage container list --output table
+Name             Lease Status    Last Modified
+---------------  --------------  -------------------------
+cluster-configs  unlocked        2020-10-06T21:12:36+00:00
+```
+
```

Comment:
```suggestion

Set the env var `KOPS_STATE_STORE` to the container name URL using kOps' `azureblob://` protocol.
The URL may include a path within the container.
kOps stores all of its cluster configuration within this path.

```bash
export KOPS_STATE_STORE=azureblob://cluster-configs
```
```

The Github suggestion feature is botching my triple backticks above but you get the idea :)